### PR TITLE
Vietnamese language support

### PR DESCRIPTION
**Pull request by NhanAZ**

### CODE REVIEW

File: resources/languages/vi_VN.ini [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+microgame.winners2="§6{%winners_count}§a/§6{%players_count} §angười chơi chiến đã chiến thắng màn này."
+microgame.success="§aTHÀNH CÔNG"
+microgame.failed="§cTHẤT BẠI"
+microgame.worth="§l§eMàn này có giá trị §6{%points} §eđiểm."
+microgame.felloffplatform="§cBạn đã rơi khỏi platform!"
+microgame.topkiller="§8{%player} §eđã giết §6{%kills_count} §engười chơi khác"
+microgame.failedtosurvive="§cBạn đã không thể sống sót."
+microgame.youwereslain="§cBạn đã bị giết bởi {%player}!"
+
+microgame.item.powerstick="§r§eGậy Sức Mạnh"
```
Comment (IvanCraft623) [inline resources/languages/vi_VN.ini:91]: sus

**Comment by IvanCraft623** [comment]
New translations have been added and one has been renamed, see: e1ccc238e5d4fed20301a269a260be2b79bfbe27

**Comment by NhanAZ** [comment]
> New translations have been added and one has been renamed, see: [e1ccc23](https://github.com/LatamPMDevs/Minerware/commit/e1ccc238e5d4fed20301a269a260be2b79bfbe27)

I'll add it tomorrow morning. Now I need to go to sleep. 02:40 AM...

**Comment by NhanAZ** [comment]
Maybe I should open another pull request to avoid the conflict

**Pull request closed without merging**